examples/lusid/run-valuation/2-create-portfolio.sql

```diff
@@ -0,0 +1,14 @@
+-- Defining scope and code variables
+@@portfolioScope = select 'ibor';
+@@portfolioCode1 = select 'EQUITY_UK';
+
+-- Define the portfolio data
+@create_portfolio = select 'Transaction' as PortfolioType, @@portfolioScope as PortfolioScope, @@portfolioCode1 as
+   PortfolioCode, @@portfolioCode1 as DisplayName, '' as Description, #2000-01-01# as Created, ''
+   as SubHoldingKeys, 'GBP' as BaseCurrency;

```

Comment:
Very minor thing, would it be better to have each value on a new line to match the formatting of the other files in the folder?